syntaxes/terraform.tmGrammar.json

```diff
@@ -638,19 +638,15 @@
 					"name": "keyword.operator.accessor.terraform"
 				}
 			},
-			"end": "\\w+(\\[[0-9\\*]+\\])?",
+			"end": "[[:alpha:]][[:alnum:]_-]*(\\[[0-9\\*]+\\])?",
```

Comment:
Can you add a `"comment"` line here explaining the regex? For future us so when we come back to this we know what this is :)

Reply:
Good idea, I've updated the comments in the attribute access section